lib/inflow_web/controllers/manifests_controller.ex

```diff
@@ -1,9 +1,13 @@
 defmodule InflowWeb.ManifestsController do
   use InflowWeb, :controller
   alias Inflow.{Import, Import.Manifest}
+  @gravity_api Application.get_env(:inflow, :gravity_api)
 
-  def index(conn, _params) do
-    render(conn, "index.html", manifests: Import.list_manifests())
+  def index(conn, %{"partner_id" => partner_id}) do
+    partner = @gravity_api.get!("/api/v1/partner/#{partner_id}").body
```

Comment:
I can't tell. Does this `get` happen every time this page is accessed, even if it's in the session?

Reply:
yes, we don't add `partner` to the session though, we only add `partner_id`.